src/Microsoft.AspNet.Diagnostics/ErrorPageOptions.cs

```diff
@@ -23,5 +25,11 @@ public ErrorPageOptions()
         /// source code referenced by the exception stack trace is present on the server.
         /// </summary>
         public int SourceCodeLineCount { get; set; }
+
+        /// <summary>
+        /// Provides files containing source code which is used to display the line(s) of exception.
+        /// if not provided, <see cref="PhysicalFileProvider"/> is used by default.
```

Comment:
Move this bit into `<remarks>`: `If <c>null</c> <see cref="ErrorPageMiddleware" /> will use <see cref="PhysicalFileProvider"/>.`